### ISSUE TITLE
Adding a role for X11 windows

### ISSUE THREAD

**Issue by cdosborn** [koekeishiya/kwm#173]
I'm running an application using XQuartz that will not tile (it's the suckless shell `st`). Below is the output of running the Accessibility Inspector. Could I get some guidance on what role should be added? 

*This might be more appropriate as a kwm wiki entry as opposed to an issue.

![screen shot 2016-02-21 at 1 46 07 pm](https://cloud.githubusercontent.com/assets/3847314/13205271/0b5035c6-d8a2-11e5-9076-356e755156ed.png)

**Comment by koekeishiya** [comment]
I have confirmed through `kwmc read windows` that applications running inside XQuartz do have a unique window id, and they are "owned by" XQuartz. 

As far as I know, it is not possible to create an AXUIElementRef for windows that run through XQuartz, and this is the reason why Kwm cannot interact with them.

**Comment by theli-ua** [comment]
While X11 windows are no accessibilityWindows you can get information about them and control them via X11 itself, for example this is how ShiftIt works with them:
https://github.com/fikovnik/ShiftIt/blob/master/ShiftIt/X11WindowDriver.m

While I do understand that would require efforts to make X11 windows behave within KWM 
this would be a really great feature addition.
